### ISSUE TITLE
Thanks for your work

### ISSUE THREAD

**Issue by kant2002** [PrismLibrary/Prism#1729]
### Description

Even I am not user of Prism library right now, I want to thank you for supporting it. I think this is excellent UI navigation library for complex applications and I somewhat resonate with your feelings. I go to create this issue from following link: https://twitter.com/jasonbock/status/1104513508174192640?s=21
Just so you know from where I am coming! Please know that some people do care!

### Steps to Reproduce

1. Submit issue
2. Do not invest in the code
3. Blame contributor

### Expected Behavior

Create PR with fix.

**Comment by lewcianci** [comment]
Yeah I agree, thanks everyone.  PRISM is easily a piece of software that could easily be charged for, and it supports such a wide range of platforms (like UWP/iOS/Android/WPF etc).  I write an app for a group of users and I don't know what I'd do without PRISM.

PRISM makes me be a better developer and has taught me the MVVM pattern for phone apps. It's a really high quality piece of software that is clearly made better by the teams incredible knowledge and resourcefulness.  I downloaded the PRISM template pack again today for vs 2019 and it has something like 350k downloads.  Clearly it's a popular framework and a lot of people would be straight up lost without it.

Unfortunately it is the negative voices that are often heard the loudest and the ones that have the most impact on us (I know as a maintenance developer for a wide range of apps at work)...  Just reflect on the sheer amount of use that Prism sees and I hope you will recognise, like I did, that the people who insult the framework/contributors view this is as a way to get what they want.  And that's more of a reflection on them then it is on you.
